dataactbroker/handlers/accountHandler.py

```diff
@@ -2,6 +2,7 @@
 from threading import Thread
 import re
 import time
+from flask import session
```

Comment:
Is this the same info as line 1 (`from flask import session as flaskSession`)? Reading through the code, it seemed like `session` and `flaskSession` are one in the same. If that's true, I think it would be better to stick with one to avoid future confusion.